spec/unit/provider_resolver_spec.rb

```diff
@@ -50,6 +54,51 @@
     allow(resource).to receive(:is_a?).with(Chef::Resource).and_return(true)
   end
 
+  def self.on_platform(platform, *tags,
+    platform_version: '11.0.1',
```

Comment:
The others are as well, just they are defaulted to other parameters (see the call later on where we use ||)